site/book/02-concepts/01-packages.md

```diff
@@ -1 +1,59 @@
-Incl. package lifecycle/workflow
+A kpt package is a bundle of configuration _data_. It is represented as a directory tree containing
+KRM resources using YAML as the file format.
+
+A package is explicitly declared using a file named `Kptfile` containing a KRM resource of kind
+`Kptfile`. The Kptfile contains metadata about the package and is just a regular resource in the
+YAML format.
+
+Just as directories can be nested, a package can contain another package, called a
+_subpackage_.
+
+Let's take a look at an example:
+
+```shell
+$ kpt pkg get https://github.com/GoogleContainerTools/kpt.git/package-examples/wordpress@next \
+wordpress
```

Comment:
Since directory is optional, you can skip it. You can compare it with git clone.

`kpt pkg get https://github.com/GoogleContainerTools/kpt.git/package-examples/wordpress@next`